tests/unit_tests/test_pychop.py

```diff
@@ -128,17 +138,35 @@ def test_fermi_invalid_e_init(e_init, mari_data: tuple[PyChopModelDataFermi, PyC
 @pytest.mark.parametrize('chopper_frequency',
                          [[59.99999, 60], [-0.048] * 2, [-np.inf, 0], [120, 300.00017], [np.inf, np.inf],
                           [300, np.nan], [60.5, 60], [180, 67.5], [600, 600], [130, 130]])
-def test_nonfermi_invalid_chopper_frequency(chopper_frequency, cncs_data: PyChopModelDataNonFermi):
+def test_cncs_invalid_chopper_frequency(chopper_frequency, cncs_data: PyChopModelDataNonFermi):
     with pytest.raises(InvalidInputError) as e:
-        PyChopModelNonFermi(cncs_data, chopper_frequency=chopper_frequency)
+        PyChopModelCNCS(cncs_data, resolution_disk_frequency=chopper_frequency[0], fermi_frequency=chopper_frequency[1])
 
     assert 'The provided chopper frequency' in str(e.value)
 
 
-@pytest.mark.parametrize('e_init', [-5, -0.00048, -np.inf, 2000.1, np.inf, 13554.1654, np.nan])
-def test_nonfermi_invalid_e_init(e_init, cncs_data: PyChopModelDataNonFermi):
+@pytest.mark.parametrize('e_init', [-5, -0.00048, -np.inf, 80.1, np.inf, 13554.1654, np.nan])
+def test_cncs_invalid_e_init(e_init, cncs_data: PyChopModelDataNonFermi):
+    with pytest.raises(InvalidInputError) as e:
```

Comment:
Use `with pytest.raises(InvalidInputError, match="The provided incident energy"):` instead of separate assertion.